### PR TITLE
WIP:Feature#40 31/35/766

### PR DESCRIPTION
**Pull request by varsha-22**
Description:
Generic Textfield component with error message, disabled and active input etc.
Image slider with random auto image slides etc.
Select field component with default value and options etc.
Radiogroup component and select value etc.
Submit Button with disabled value until form filed get filled and alert on submit etc.
Cancel Button and Form Validation using 'yup' is in WIP

### CODE REVIEW

**Review by pinku4444 — COMMENTED** [review]
Use external css and remove all commented code.

File: src/components/SelectField/SelectField.jsx [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+  //   });
+  //   this.state.isEnabled = true;
+  // }
+
+ 
+
+  renderSwitch() {
+    // const { rgrpid1, rgrpid2 } = this.props;
+
+    // console.log(rgrpid1);
```
Comment (pinku4444) [inline src/components/SelectField/SelectField.jsx:76]: Remove all console.log and commented code.

File: src/components-03-10-2019/TextField/TextFieldbackup.jsx [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+         'errmsg' : ''
+      };
+   }
+   textChangeHandler = (event) => {
+      let nam = event.target.name;
+      let val = event.target.value;
+      let err = '';
+      if(nam === 'age'){
+         if(val !== "" && !Number(val)){
+            err = <strong style ={{color: 'red'}}> Your age must be a number</strong>
```
Comment (pinku4444) [inline src/components-03-10-2019/TextField/TextFieldbackup.jsx:39]: Remove all inline css.

File: src/components-03-10-2019/SelectField/SelectField.jsx [hunk 3]
```diff
@@ -0,0 +1,89 @@
+import React, { Component } from 'react'
+import RadioGroups from '.././RadioGroup/RadioGroup';
+const selectStyle = {
```
Comment (pinku4444) [inline src/components-03-10-2019/SelectField/SelectField.jsx:3]: Use external css.

**Comment by varsha-22** [comment]
> Use external css and remove all commented code.

Done

**Pull request merged**